netflix-roulette/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const path = require("path")
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const WebpackNotifierPlugin = require('webpack-notifier')
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+
+module.exports = (env, argv) => {
+    const modeEnv = argv.mode || 'development'
+    const isProd = modeEnv === 'production'
+
+    let optimizations = {
+        splitChunks: {
+            cacheGroups: {
+                vendors: {
```

Comment:
There's, probably, not much value in separating vendors chunk for development mode, so, maybe, should be included into production version of the config only.